web/app/themes/dp-timber-theme/views/footer.twig

```diff
@@ -1 +1,2 @@
-Copyright {{"now"|date('Y')}}
\ No newline at end of file
+Copyright {{"now"|date('Y')}}
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
Why do we need this now?

Reply:
No we don't need this, I taught having an simpler alternative way, I will remove it